xml/System.Net.Security/TlsCipherSuite.xml

```diff
@@ -39,7 +46,7 @@
       </ReturnValue>
       <MemberValue>4869</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Represents the TLS_AES_128_CCM_8_SHA256 cipher suite.</summary>
```

Comment:
not checking anything after here, let me know if there is something there to check